_benchmark/user-guide/install-and-configure/configuring-benchmark.md

```diff
@@ -159,6 +160,53 @@ This section defines how OpenSearch versions are distributed.
 | :---- | :---- | :---- |
 | `release.cache` | Boolean | Determines whether newly released OpenSearch versions should be cached locally. |
 
+## Running OpenSearch Benchmark with AWS Signature Version 4
+
+OpenSearch Benchmark supports AWS Signature Version 4 authentication.  To run OpenSearch Benchmark with AWS Signature Version 4, you need to set up an [AWS Identity and Access Management (IAM) user or role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html) and provide it access to the OpenSearch cluster using AWS Signature Version 4 authentication. 
+
+The choice between an IAM role or user depends on your test cluster's access management requirements. For more information on deciding whether to use a role or user, see [When to create an IAM user (instead of a role)](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html#id_which-to-choose).
```

Comment:
```suggestion
Whether to use an IAM role or user depends on your test cluster's access management requirements. For more information about whether to use an IAM role or user, see [When to create an IAM user (instead of a role)](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html#id_which-to-choose).
```